bom/pom.xml

```diff
@@ -36,10 +36,10 @@
         <hdrHistogramVersion>2.1.8</hdrHistogramVersion>
         <latencyUtilsVersion>2.0.3</latencyUtilsVersion>
         <!-- Neo4j supports different versions of Cypher -->
-        <neo4jCypherCompiler19Version>2.0.5</neo4jCypherCompiler19Version>
-        <neo4jCypherCompiler22Version>2.2.9</neo4jCypherCompiler22Version>
+        <neo4jCypherCompiler23Version>2.3.4</neo4jCypherCompiler23Version>
+        <neo4jCypherCompiler30Version>${neo4jVersion}</neo4jCypherCompiler30Version>
```

Comment:
These are Neo4j dependecies and an error is thrown if hey are not on the classpath